nvflare/lighter/impl/docker_launcher.py

```diff
@@ -0,0 +1,205 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import copy
+import json
+import os
+import shutil
+
+import yaml
+
+from nvflare.app_opt.job_launcher.docker_launcher import ClientDockerJobLauncher, ServerDockerJobLauncher
```

Comment:
Since DockerLauncher depends on "import docker", if we do this import here, are we enforcing all the users must install the "docker" package? @yhwen @chesterxgchen @IsaacYangSLA

Reply:
Yes, if user wants to build the DockerLauncher package, they need to have docker installed to use.

Reply:
Got it, if they don't want to use this builder they don't need to install docker to run NVFlare right?

Reply:
tested. This is not an issue.

Reply:
@yhwen have already tested this, so no issue with this.
If user not use this in their project.yml then without docker installation, provision still work.